### PR TITLE
Bug548495 - TEE: "new" Builds not showing up in the builds page

### PR DESCRIPTION
**Pull request by AlexRukhlin**
The only we can do now is to clarify in TEE labels that they have deas with XAML builds.

### CODE REVIEW

**Comment by DavidStaheli** [comment]
Looks good to me

File: source/com.microsoft.tfs.client.common/src/com/microsoft/tfs/client/common/git/utils/GitHelpers.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public static boolean isEGitInstalled(final boolean activateEGit) {
         } else if (activateEGit) {
             return activateBundle(egit_core) && activateEGitUI();
         } else {
-            return false;
```
Comment (jpricketMSFT) [inline source/com.microsoft.tfs.client.common/src/com/microsoft/tfs/client/common/git/utils/GitHelpers.java]: It would be good here to have a comment that explains how we got into this else clause and why we should return true.

File: source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/link/TeamExplorerBaseNavigationLink.java [hunk 2]
```diff
@@ -27,6 +27,7 @@ public final void clicked(
         final TeamExplorerNavigator navigator,
         final TeamExplorerNavigationItemConfig parentNavigationItem) {
 
+        onClick(shell, context, navigator, parentNavigationItem);
```
Comment (jpricketMSFT) [inline source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/link/TeamExplorerBaseNavigationLink.java]: Is this related to the fix?

**Comment by jpricketMSFT** [comment]
Approved with suggestions/questions

File: source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/link/TeamExplorerBaseNavigationLink.java (hunk 2 above)
Comment (AlexRukhlin) [inline source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/link/TeamExplorerBaseNavigationLink.java]: It's related to Build navigation items (and some other as well). I put it into a separate commit.
